content/client-server-api/_index.md

```diff
@@ -1195,7 +1197,7 @@ with just the session ID:
 }
 ```
 
-##### Example
+**Example**
```

Comment:
Same here, I downgraded this to a non-heading, because I feel like it doesn't need to appear in the ToC.